core/src/main/java/io/grpc/ClientCall.java

```diff
@@ -254,6 +255,24 @@ public void setMessageCompression(boolean enabled) {
     // noop
   }
 
+  /**
+   * Defers flushing the messages written using {@link #sendMessage(Object)} to the
```

Comment:
I think this should mention "or the cork is closed" to make it more obvious.

How about:
> Defers flushing the messages written using sendMessage() until the cork is closed or as necessary to prevent excessive buffering.